typespec-tests/tsp/examples/2022-06-01-preview/FlattenOp_SendLong.json

```diff
@@ -8,8 +8,7 @@
       "user": {
         "user": "myOptionalUser"
       },
-      "input": "myRequiredInput",
-      "dataInt": 11,
+      "input": "myInput",
```

Comment:
`myRequiredInput` -> `myInput` is not intended.. Just don't want to regen the code again..